src/Umbraco.Core/Models/IFile.cs

```diff
@@ -40,6 +40,14 @@ public interface IFile : IAggregateRoot
         string Content { get; set; }
 
         /// <summary>
+        /// Indicates that the file should use whatever content already
+        /// exists on the filesystem which manages the file, bypassing content
+        /// management entirely.
+        /// </summary>
+        /// <remarks>Only for the next save. Is resetted when the file is saved.</remarks>
+        void UseExistingContent();

```

Comment:
Can we leave this change off of this template for now ? For the 7.5 release we can add it back but we cannot break interfaces if we want to include this PR for 7.4.1.